README.md

```diff
@@ -13,7 +13,7 @@ For questions and support, please visit the [Network Canvas User Community](http
 ### Prerequisites
 
 - [Node.js](https://nodejs.org/en/) (v14.21.3)
-- [npm](https://www.npmjs.com/) (v8.19.4)
+- [npm](https://www.npmjs.com/) (v8.3.2)
```

Comment:
I think this was correct previously - if package.json says 8.3.2 I would suggest updating that, instead.

Reply:
Sure, will do sir